src/switchroot/ostree-prepare-root.c

```diff
@@ -174,17 +174,19 @@ pivot_root(const char * new_root, const char * put_old)
 int
 main(int argc, char *argv[])
 {
-  const char *root_mountpoint = NULL;
+  const char *root_mountpoint = NULL, *root_arg = NULL;
   char *deploy_path = NULL;
   char srcpath[PATH_MAX];
   struct stat stbuf;
 
   if (argc < 2)
-    root_mountpoint = "/";
+    root_arg = "/";
   else
-    root_mountpoint = argv[1];
+    root_arg = argv[1];
 
-  root_mountpoint = realpath (root_mountpoint, NULL);
+  root_mountpoint = realpath (root_arg, NULL);
```

Comment:
Hmm...one thing I started wondering; why are we invoking `realpath` at all here?   We already had another invocation inside `resolve_deploy_path()` which dates to  bb6eedfb258d3001f61c42c7e920c03dae2bdc1a which I probably wrote on a netbook while riding a dinosaur or something.

Is it just cosmetic, or did I have an actual goal there?  I don't remember...might be interesting to try taking the realpath invocations out.